request.js

```diff
@@ -48,9 +48,6 @@ module.exports = function createClient(defaults) {
 	if (typeof defaults === 'string') {
 		defaults = {api_key: defaults};
 	}
-	if (!defaults.api_key) {
```

Comment:
I'm removing this because all of the API methods themselves have this built in already, so this is redundant